httpclient/README.md

```diff
@@ -1,9 +1,11 @@
-# HTTP client capability provider
+# HTTP Client Capability Provider
 
-This capability provider implements the "wasmcloud:httpclient" capability
-contract using the rust 'reqwest' library
+This capability provider implements the `wasmcloud:httpclient` capability contract using the Rust [reqwest](https://docs.rs/reqwest) library.
```

Comment:
This capability provider implements the `wasmcloud:httpclient` capability contract, and enables an actor to make outgoing http(s) requests. It is implemented in Rust using the [reqwest](https://docs.rs/reqwest) library.